### ISSUE TITLE
A11y_SQL Azure Data Studio_Import Extension - Import ExtensionModify Columns_ScreenReader: There are more than 6 list items present in the drop down of column 2 yet they all are announced as 6 of 7.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#12977]
“[Check out Accessibility Insights](https://accessibilityinsights.io/)! – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**: #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Oct2020;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.3.1;#Benchmark;#NVDA;#AILimited;#AINotInScope;

**Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup
Windows Version: Windows10
Screen reader: NVDA2020.3

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to the connected server and press enter key, use down key and select any database keep_MDW Right click on/ open the context menu for a database and select Data-tier application wizard from the context menu
4. Add data in specify input file screen and tab to "next" button and press enter.
5. Press enter on next button.
6. Launch screen reader as NVDA and tab to drop down in column 2 and press enter on it , navigate to each item using down key arrow. 

**Actual:**
On navigating to the first item in the list it is announced as 1 of 6 the 6th item is announced as 6 of 7 and the 7th and rest of the items are announced as 6 of 7.

**Issue not observed with JAWS**

**Expected:**
On navigating to the first item the 1st item should be announced as 1 of (total number of items in the list) and thus the rest should follow.

**User Impact:**
The screen reader users will get the incorrect list numbers.

**Attachment:**

[#12977_A11y_SQL Azure Data Studio_Import ExtensionModify Columns_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/5401656/12977_A11y_SQL.Azure.Data.Studio_Import.ExtensionModify.Columns_ScreenReader.zip)

**Comment by aasimkhan30** [comment]
The tree is virtualized.

**Comment by fsteffi** [comment]
hi @alanrenmsft has the item "Data-tier application wizard " been removed
![image](https://user-images.githubusercontent.com/24580570/220045350-8a5d44c8-c9b3-4a49-9de6-acf968479c0f.png)


GithubTags:#Regressed:02-20-23;

**Comment by fsteffi** [comment]
fixed on 
Version: 1.42.0-insider
Commit: 872095ed849537d6c96373eedcd187b3f707c460
Date: 2023-02-23T06:29:37.626Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.25300

GithubTags:#closed;
[12977.webm](https://user-images.githubusercontent.com/24580570/220863860-b0614ae3-c75d-43ae-b16b-58f5be7a43b9.webm)
